docs/reference/settings.rst

```diff
@@ -315,6 +315,15 @@ Specifies the number of items per page shown when viewing an image's usage (see
 
 Specifies the number of images shown per page in the image chooser modal.
 
+``WAGTAILIMAGES_RENDITION_STORAGE``
+-----------------------------------
+
+.. code-block:: python
+
+    WAGTAILIMAGES_RENDITION_STORAGE = None
+
+Specifies an alternate storage path for image renditions.
```

Comment:
```suggestion
Allows renditions to be stored in a distinct filesystem location from other uploaded files outside of `MEDIA_ROOT`.
For example, 'public' would put store all renditions in a `/public` folder configured at the server level.
```

Just copied / reworded the content from the original issue to give a bit more information, feel free to revise.